cardio_crumble.opam

```diff
@@ -13,8 +13,6 @@ depends: [
   "ocaml"
   "dune" {>= "3.3"}
   "odoc" {with-doc}
-  "portmidi"
```

Comment:
I don't think this should go away?

Reply:
(I did not review the rest yet, will take a look maybe later today or this week end! But this is great, thank you very much!)

Reply:
Thanks for the review @Engil !
It seems like I deleted this accidently xD while going through all the files, and checking dependencies etc.

Reply:
@Engil , just checked, this is weird
When I run,
```
dune build
```
It removes `portmidi` automatically. Is this not being used in the project anymore?

Reply:
> The dependencies should be added to dune-project, not the .opam file. Dune seems to be handling dependencies by itself

We've specified that we want Dune to do so by having `(generate_opam_files true)` in the `dune-project`.

@AryanGodara, do you want to fix this for us by adding the two dependencies to the `dune-project` file (and re-adding them to the `.opam`-file)? You could have a look at the [`dune-project` file of the portmidi library](https://github.com/mbacarella/portmidi/blob/main/dune-project#L22-L33) to see the syntax. No pressure! I'm happy to do it myself if you prefer to focus on the scale task.

Reply:
No I'd definitely like to work on this issue @pitag-ha !!
Should I open a separate issue for this? Or increase the scope of this current PR?

Reply:
> No I'd definitely like to work on this issue @pitag-ha !!

Nice!

> Should I open a separate issue for this? Or increase the scope of this current PR?

I leave that up to you. The cleaner version would be to open a separate PR, merge it, and then rebase this branch over the branch of the other PR. However, as you've said, this change is minimal, so mixing the things here is ok as well. If you think otherwise about git history pedantry, @Engil, let us know!